### DISCUSSION TITLE
Hierarchical Federated Learning

### DISCUSSION THREAD

**Discussion by dimalkavindu** [APPFL/APPFL#161] [Q&A]
Does APPFL support Hierarchical Federated Learning  ? (Introducing multiple layers of model aggregation E.g. Edge level aggregation)

**Comment by kibaekkim** [comment]
It does not support a hierarchical FL. If you are interested in implementing one in our framework, our team would be happy to support you.

Kibaek
________________________________
From: Dimal Kavindu ***@***.***>
Sent: Saturday, February 3, 2024 8:04:05 AM
To: APPFL/APPFL ***@***.***>
Cc: Subscribed ***@***.***>
Subject: [APPFL/APPFL] Hierarchical Federated Learning (Discussion #161)


Does APPFL support Hierarchical Federated Learning ? (Introducing multiple layers of model aggregation E.g. Edge level aggregation)
